src/components/menus/stage/StagedFilterMenu.tsx

```diff
@@ -1,7 +1,10 @@
+import { FilterParameter } from '@circleci/circleci-config-sdk/dist/src/lib/Components/Parameters/types';
```

Comment:
import the config sdk as `*` for proper linking

Reply:
Use destructuring here